stdlib/public/core/SIMDVector.swift

```diff
@@ -1407,3 +1473,152 @@ where T: SIMD, T.Scalar: FloatingPoint {
   }
   return result
 }
+
```

Comment:
Add `// SWIFT_ENABLE_TENSORFLOW` above.

Reply:
I should do this for each extension I added right?